# Finnish (Finland) translations for keystone.
# Copyright (C) 2013 ORGANIZATION
# This file is distributed under the same license as the keystone project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version:  Keystone\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/keystone\n"
"POT-Creation-Date: 2013-07-11 17:05+0000\n"
"PO-Revision-Date: 2013-05-17 16:06+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Finnish (Finland) "
"(http://www.transifex.com/projects/p/openstack/language/fi_FI/)\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.6\n"

#: keystone/clean.py:23
#, python-format
msgid "%s cannot be empty."
msgstr ""

#: keystone/clean.py:25
#, python-format
msgid "%(property_name)s cannot be less than %(min_length)s characters."
msgstr ""

#: keystone/clean.py:29
#, python-format
msgid "%(property_name)s should not be greater than %(max_length)s characters."
msgstr ""

#: keystone/clean.py:36
#, python-format
msgid "%(property_name)s is not a %(display_expected_type)s"
msgstr ""

#: keystone/test.py:106
#, python-format
msgid "Failed to checkout %s"
msgstr ""

#: keystone/assignment/backends/ldap.py:110
#, python-format
msgid "Expected dict or list: %s"
msgstr ""

#: keystone/assignment/backends/ldap.py:456
#: keystone/identity/backends/ldap.py:433
#, python-format
msgid "Role %s not found"
msgstr ""

#: keystone/assignment/backends/sql.py:244
#, python-format
msgid "Cannot remove role that has not been granted, %s"
msgstr ""

#: keystone/auth/controllers.py:72
#, python-format
msgid "Project is disabled: %s"
msgstr ""

#: keystone/auth/controllers.py:78 keystone/auth/plugins/password.py:38
#, python-format
msgid "Domain is disabled: %s"
msgstr ""

#: keystone/auth/controllers.py:84 keystone/auth/plugins/password.py:44
#, python-format
msgid "User is disabled: %s"
msgstr ""

#: keystone/auth/controllers.py:259
msgid "Scoping to both domain and project is not allowed"
msgstr ""

#: keystone/auth/controllers.py:262
msgid "Scoping to both domain and trust is not allowed"
msgstr ""

#: keystone/auth/controllers.py:265
msgid "Scoping to both project and trust is not allowed"
msgstr ""

#: keystone/auth/controllers.py:325
#, python-format
msgid "Unable to lookup user %s"
msgstr ""

#: keystone/auth/controllers.py:355
msgid "User not found"
msgstr ""

#: keystone/auth/token_factory.py:79
msgid "User have no access to project"
msgstr ""

#: keystone/auth/token_factory.py:94
msgid "User have no access to domain"
msgstr ""

#: keystone/auth/token_factory.py:311 keystone/token/controllers.py:119
msgid "Unable to sign token."
msgstr ""

#: keystone/auth/token_factory.py:314 keystone/token/controllers.py:122
#, python-format
msgid "Invalid value for token_format: %s.  Allowed values are PKI or UUID."
msgstr ""

#: keystone/catalog/core.py:38
#, python-format
msgid "Malformed endpoint %(url)s - unknown key %(keyerror)s"
msgstr ""

#: keystone/catalog/core.py:43
#, python-format
msgid ""
"Malformed endpoint %(url)s - unknown key %(keyerror)s(are you missing "
"brackets ?)"
msgstr ""

#: keystone/catalog/core.py:49
#, python-format
msgid ""
"Malformed endpoint %s - incomplete format                   (are you "
"missing a type notifier ?)"
msgstr ""

#: keystone/catalog/backends/templated.py:109
#, python-format
msgid "Unable to open template file %s"
msgstr ""

#: keystone/common/cms.py:26
#, python-format
msgid "Verify error: %s"
msgstr ""

#: keystone/common/cms.py:118
msgid ""
"Signing error: Unable to load certificate - ensure you've configured PKI "
"with 'keystone-manage pki_setup'"
msgstr ""

#: keystone/common/cms.py:122
#, python-format
msgid "Signing error: %s"
msgstr ""

#: keystone/common/config.py:89
#, python-format
msgid "Unable to locate specified logging config file: %s"
msgstr ""

#: keystone/common/config.py:107
msgid "Invalid syslog facility"
msgstr ""

#: keystone/common/controller.py:18
#, python-format
msgid "RBAC: Authorizing %(action)s(%(kwargs)s)"
msgstr ""

#: keystone/common/controller.py:25
msgid "RBAC: Invalid token"
msgstr ""

#: keystone/common/controller.py:35 keystone/common/controller.py:56
msgid "RBAC: Invalid user"
msgstr ""

#: keystone/common/controller.py:41
msgid "RBAC: Proceeding without project"
msgstr ""

#: keystone/common/controller.py:61
msgid "RBAC: Proceeding without tenant"
msgstr ""

#: keystone/common/controller.py:91 keystone/common/controller.py:142
msgid "RBAC: Bypassing authorization"
msgstr ""

#: keystone/common/controller.py:100 keystone/common/controller.py:140
msgid "RBAC: Authorization granted"
msgstr ""

#: keystone/common/controller.py:130
#, python-format
msgid "RBAC: Adding query filter params (%s)"
msgstr ""

#: keystone/common/controller.py:318
msgid "Invalid token in normalize_domain_id"
msgstr ""

#: keystone/common/utils.py:233
msgid ""
"Error setting up the debug environment. Verify that the option --debug-"
"url has the format <host>:<port> and that a debugger processes is "
"listening on that port."
msgstr ""

#: keystone/common/wsgi.py:162
#, python-format
msgid "arg_dict: %s"
msgstr ""

#: keystone/common/wsgi.py:188
#, python-format
msgid "Authorization failed. %(exception)s from %(remote_addr)s"
msgstr ""

#: keystone/common/wsgi.py:431
msgid "The resource could not be found."
msgstr ""

#: keystone/common/environment/__init__.py:37
#, python-format
msgid "Environment configured as: %s"
msgstr ""

#: keystone/common/environment/eventlet_server.py:51
#, python-format
msgid "Starting %(arg0)s on %(host)s:%(port)s"
msgstr ""

#: keystone/common/environment/eventlet_server.py:113
msgid "Server error"
msgstr ""

#: keystone/common/ldap/core.py:79
#, python-format
msgid "Invalid LDAP deref option: %s. Choose one of: "
msgstr ""

#: keystone/common/ldap/core.py:87
#, python-format
msgid "Invalid LDAP TLS certs option: %(option). Choose one of: %(options)s"
msgstr ""

#: keystone/common/ldap/core.py:99
#, python-format
msgid "Invalid LDAP scope: %(scope)s. Choose one of: %(options)s"
msgstr ""

#: keystone/common/ldap/core.py:189
#, python-format
msgid ""
"Invalid additional attribute mapping: \"%s\". Format must be "
"<ldap_attribute>:<keystone_attribute>"
msgstr ""

#: keystone/common/ldap/core.py:195
#, python-format
msgid ""
"Invalid additional attribute mapping: \"%(item)s\". Value "
"\"%(attr_map)s\" must use one of %(keys)s."
msgstr ""

#: keystone/common/ldap/core.py:279 keystone/identity/backends/kvs.py:177
#: keystone/identity/backends/kvs.py:205
#, python-format
msgid "Duplicate name, %s."
msgstr ""

#: keystone/common/ldap/core.py:289 keystone/identity/backends/kvs.py:170
#, python-format
msgid "Duplicate ID, %s."
msgstr ""

#: keystone/common/ldap/core.py:294
#, python-format
msgid "LDAP %s create"
msgstr ""

#: keystone/common/ldap/core.py:372
#, python-format
msgid "LDAP %s update"
msgstr ""

#: keystone/common/ldap/core.py:405
#, python-format
msgid "LDAP %s delete"
msgstr ""

#: keystone/common/ldap/core.py:430
#, python-format
msgid "LDAP init: url=%s"
msgstr ""

#: keystone/common/ldap/core.py:431
#, python-format
msgid ""
"LDAP init: use_tls=%(use_tls)s\n"
"tls_cacertfile=%(tls_cacertfile)s\n"
"tls_cacertdir=%(tls_cacertdir)s\n"
"tls_req_cert=%(tls_req_cert)s\n"
"tls_avail=%(tls_avail)s\n"
msgstr ""

#: keystone/common/ldap/core.py:450
msgid "Invalid TLS / LDAPS combination"
msgstr ""

#: keystone/common/ldap/core.py:454
#, python-format
msgid "Invalid LDAP TLS_AVAIL option: %s. TLS not available"
msgstr ""

#: keystone/common/ldap/core.py:464
#, python-format
msgid "tls_cacertfile %s not found or is not a file"
msgstr ""

#: keystone/common/ldap/core.py:476
#, python-format
msgid "tls_cacertdir %s not found or is not a directory"
msgstr ""

#: keystone/common/ldap/core.py:483
#, python-format
msgid "LDAP TLS: invalid TLS_REQUIRE_CERT Option=%s"
msgstr ""

#: keystone/common/ldap/core.py:497
#, python-format
msgid "LDAP bind: dn=%s"
msgstr ""

#: keystone/common/ldap/core.py:508
#, python-format
msgid "LDAP add: dn=%(dn)s, attrs=%(attrs)s"
msgstr ""

#: keystone/common/ldap/core.py:514
#, python-format
msgid ""
"LDAP search: dn=%(dn)s, scope=%(scope)s, query=%(query)s, "
"attrs=%(attrlist)s"
msgstr ""

#: keystone/common/ldap/core.py:567
msgid ""
"LDAP Server does not support paging. Disable paging in keystone.conf to "
"avoid this message."
msgstr ""

#: keystone/common/ldap/core.py:584
#, python-format
msgid "LDAP modify: dn=%(dn)s, modlist=%(modlist)s"
msgstr ""

#: keystone/common/ldap/core.py:590
#, python-format
msgid "LDAP delete: dn=%s"
msgstr ""

#: keystone/common/ldap/core.py:595
#, python-format
msgid "LDAP delete_ext: dn=%(dn)s, serverctrls=%(serverctrls)s"
msgstr ""

#: keystone/common/ldap/fakeldap.py:146
#, python-format
msgid "FakeLdap initialize url=%s"
msgstr ""

#: keystone/common/ldap/fakeldap.py:156
#, python-format
msgid "FakeLdap bind dn=%s"
msgstr ""

#: keystone/common/ldap/fakeldap.py:163
#, python-format
msgid "FakeLdap bind fail: dn=%s not found"
msgstr ""

#: keystone/common/ldap/fakeldap.py:170
#, python-format
msgid "FakeLdap bind fail: password for dn=%s not found"
msgstr ""

#: keystone/common/ldap/fakeldap.py:175
#, python-format
msgid "FakeLdap bind fail: password for dn=%s does not match"
msgstr ""

#: keystone/common/ldap/fakeldap.py:190
#, python-format
msgid "FakeLdap add item: dn=%(dn)s, attrs=%(attrs)s"
msgstr ""

#: keystone/common/ldap/fakeldap.py:193
#, python-format
msgid "FakeLdap add item failed: dn=%s is already in store."
msgstr ""

#: keystone/common/ldap/fakeldap.py:207 keystone/common/ldap/fakeldap.py:221
#, python-format
msgid "FakeLdap delete item: dn=%s"
msgstr ""

#: keystone/common/ldap/fakeldap.py:211 keystone/common/ldap/fakeldap.py:225
#, python-format
msgid "FakeLdap delete item failed: dn=%s not found."
msgstr ""

#: keystone/common/ldap/fakeldap.py:240
#, python-format
msgid "FakeLdap modify item: dn=%(dn)s attrs=%(attrs)s"
msgstr ""

#: keystone/common/ldap/fakeldap.py:245
#, python-format
msgid "FakeLdap modify item failed: dn=%s not found."
msgstr ""

#: keystone/common/ldap/fakeldap.py:262
#, python-format
msgid "FakeLdap modify item failed: item has no attribute \"%s\" to delete"
msgstr ""

#: keystone/common/ldap/fakeldap.py:273
#, python-format
msgid ""
"FakeLdap modify item failed: item has no attribute \"%(k)s\" with value "
"\"%(v)s\" to delete"
msgstr ""

#: keystone/common/ldap/fakeldap.py:279
#, python-format
msgid "FakeLdap modify item failed: unknown command %s"
msgstr ""

#: keystone/common/ldap/fakeldap.py:281
#, python-format
msgid "modify_s action %s not implemented"
msgstr ""

#: keystone/common/ldap/fakeldap.py:300
#, python-format
msgid "FakeLdap search at dn=%(dn)s scope=%(scope)s query=%(query)s"
msgstr ""

#: keystone/common/ldap/fakeldap.py:306
msgid "FakeLdap search fail: dn not found for SCOPE_BASE"
msgstr ""

#: keystone/common/ldap/fakeldap.py:320
#, python-format
msgid "Search scope %s not implemented."
msgstr ""

#: keystone/common/sql/core.py:126
msgid "Global engine callback raised."
msgstr ""

#: keystone/common/sql/core.py:240
#, python-format
msgid "Got mysql server has gone away: %s"
msgstr ""

#: keystone/common/sql/legacy.py:188
#, python-format
msgid "Cannot migrate EC2 credential: %s"
msgstr ""

#: keystone/common/sql/migration.py:47
msgid "version should be an integer"
msgstr ""

#: keystone/common/sql/nova.py:65
#, python-format
msgid "Create tenant %s"
msgstr ""

#: keystone/common/sql/nova.py:82
#, python-format
msgid "Create user %s"
msgstr ""

#: keystone/common/sql/nova.py:91
#, python-format
msgid "Add user %(user_id)s to tenant %(tenant_id)s"
msgstr ""

#: keystone/common/sql/nova.py:100
#, python-format
msgid "Ignoring existing role %s"
msgstr ""

#: keystone/common/sql/nova.py:107
#, python-format
msgid "Create role %s"
msgstr ""

#: keystone/common/sql/nova.py:117
#, python-format
msgid "Assign role %(role_id)s to user %(user_id)s on tenant %(tenant_id)s"
msgstr ""

#: keystone/common/sql/nova.py:136
#, python-format
msgid "Creating ec2 cred for user %(user_id)s and tenant %(tenant_id)s"
msgstr ""

#: keystone/identity/controllers.py:878
#, python-format
msgid ""
"Group %(group)s not found for role-assignment - %(target)s with Role: "
"%(role)s"
msgstr ""

#: keystone/identity/backends/kvs.py:126 keystone/identity/backends/kvs.py:135
msgid "User not found in group"
msgstr ""

#: keystone/identity/backends/ldap.py:192
#, python-format
msgid ""
"Group member '%(user_dn)s' not found in '%(group_id)s'. The user should "
"be removed from the group. The user will be ignored."
msgstr ""

#: keystone/identity/backends/ldap.py:621
msgid "Changing Name not supported by LDAP"
msgstr ""

#: keystone/identity/backends/ldap.py:634
#, python-format
msgid "User %(user_id)s is already a member of group %(group_id)s"
msgstr ""

#: keystone/openstack/common/policy.py:394
#, python-format
msgid "Failed to understand rule %(rule)s"
msgstr ""

#: keystone/openstack/common/policy.py:404
#, python-format
msgid "No handler for matches of kind %s"
msgstr ""

#: keystone/openstack/common/policy.py:679
#, python-format
msgid "Failed to understand rule %(rule)r"
msgstr ""

#: keystone/policy/backends/rules.py:93
#, python-format
msgid "enforce %(action)s: %(credentials)s"
msgstr ""

#: keystone/token/controllers.py:396
#, python-format
msgid "User %(u_id)s is unauthorized for tenant %(t_id)s"
msgstr ""

#: keystone/token/controllers.py:413 keystone/token/controllers.py:416
msgid "Token does not belong to specified tenant."
msgstr ""

#: keystone/token/controllers.py:423
msgid "Non-default domain is not supported"
msgstr ""

#: keystone/token/controllers.py:431
msgid "Domain scoped token is not supported"
msgstr ""

#: keystone/token/backends/memcache.py:144
#, python-format
msgid ""
"Successful set of token-index-list for user-key \"%(user_key)s\", "
"#%(count)d records"
msgstr ""

#: keystone/token/backends/memcache.py:154
#, python-format
msgid ""
"Failed to set token-index-list for user-key \"%(user_key)s\". Attempt "
"%(cas_retry)d of %(cas_retry_max)d"
msgstr ""

#: keystone/token/backends/memcache.py:163
msgid "Unable to add token user list"
msgstr ""

#: keystone/token/backends/memcache.py:172
msgid "Unable to add token to revocation list."
msgstr ""

